sticker-utils/tensorflow/write_helper.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import tensorflow as tf
+import toml
+
+from config import DefaultConfig
+
+
+def read_shapes(args):
+    with open(args.shape_file) as shapesfile:
+        shapes = toml.loads(shapesfile.read())
+    return shapes
+
+
+def create_graph(config, model, args):
+    shapes = read_shapes(args)
+    graph_filename = args.output_graph_file
+
+    gpuopts = tf.GPUOptions(per_process_gpu_memory_fraction=0.3)
```

Comment:
Not for this PR, but in a future PR, we should change this to always run on the CPU. There is not much use running this on a GPU.